tc/sgx/trusted_worker_manager/enclave/singleton/work_order_enclave.cpp

```diff
@@ -13,7 +13,7 @@
  * limitations under the License.
```

Comment:
Can we rename work_order_enclave.cpp -> work_order_enclave_singleton.cpp

Reply:
Done